app/experimenter/nimbus-ui/src/components/FormOverview/index.test.tsx

```diff
@@ -0,0 +1,254 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import {
+  render,
+  screen,
+  act,
+  fireEvent,
+  cleanup,
+} from "@testing-library/react";
+import FormOverview from ".";
+import { mockExperimentQuery } from "../../lib/mocks";
+import { getExperiment } from "../../types/getExperiment";
+
+describe("FormOverview", () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  it("renders as expected", async () => {
+    render(<Subject />);
+    await act(async () => {
+      expect(screen.getByTestId("FormOverview")).toBeInTheDocument();
+    });
+  });
+
+  it("calls onCancel when cancel clicked", async () => {
```

Comment:
Oh hey, we're also almost done with EXP-462 now

Reply:
Oooh I am so tempted to finish it, but it's not in the epic so I will resist for now